config/jobs/kubernetes-csi/gen-jobs.sh

```diff
@@ -24,26 +24,28 @@ base="$(dirname $0)"
 # irrelevant because the prow.sh script will pick a suitable KinD
 # image or build from source.
 k8s_versions="
-1.27
 1.28
 1.29
+1.30
+1.31
 "
 
 # All the deployment versions we're testing.
 deployment_versions="
-1.27
 1.28
 1.29
+1.30
+1.31
 "
 
 # The experimental version for which jobs are optional.
-experimental_k8s_version="1.29"
+experimental_k8s_version="1.30"
 
 # The latest stable Kubernetes version for testing alpha jobs
-latest_stable_k8s_version="1.28"
+latest_stable_k8s_version="1.31"
 
 # Tag of the hostpath driver we should use for sidecar pull jobs
-hostpath_driver_version="v1.12.1"
+hostpath_driver_version="v1.14.1"
```

Comment:
We'll cut 1.15.0 after the changelog PR is merged: https://github.com/kubernetes-csi/csi-driver-host-path/pull/555

Reply:
sounds good. will update the PR, which we will be holding anyway until that happens